docs/changelog.md

```diff
@@ -9,6 +9,9 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/firefox-android/blob/main/android-components/plugins/dependencies/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/firefox-android/blob/main/android-components/.config.yml)
 
+* **service-pocket**
+  * 🌟 Exposed `site` parameter for sponsored Pocket stories request. [Bug 1811531](https://bugzilla.mozilla.org/show_bug.cgi?id=1811531).
```

Comment:
This would come to expose other properties.
Also in that other ticket we're not exposing `site` but rather allow specifying a site id to use when querying sponsored stories.
It could help to add a bit more info about what each property helps with.